test/rollup-main-L1.test.js

```diff
@@ -7,7 +7,7 @@ const Scalar = require("ffjavascript").Scalar;
 const RollupDB = require("@hermeznetwork/commonjs").RollupDB;
 const Account = require("@hermeznetwork/commonjs").HermezAccount;
 const Constants = require("@hermeznetwork/commonjs").Constants;
-const float16 = require("@hermeznetwork/commonjs").float16;
+const float40 = require("@hermeznetwork/commonjs").float40;
 
 const { depositTx, assertBatch } = require("./helpers/helpers");
 
```

Comment:
Change edge cases to use maximum loadAmountF and amountF allowed:
- https://github.com/jbaylina/circuits-1/blob/master/test/rollup-main-L1.test.js#L150
- https://github.com/jbaylina/circuits-1/blob/master/test/rollup-main-L1.test.js#L190
- https://github.com/jbaylina/circuits-1/blob/master/test/rollup-main-L1.test.js#L201